### DISCUSSION TITLE
Welcome to lite-rpc Discussions!

### DISCUSSION THREAD

**Discussion by aniketfuryrocks** [blockworks-foundation/lite-rpc#110] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by airose007** [comment]
Hello, is it normal for transactions to run slow? I normally get it back at much higher speed when using quicknode rpc

**Comment by leonardo3dp** [comment]
Wouldn't it be possible to redirect the disabled query methods to another rpc?

**Comment by leonardo3dp** [comment]
When I arrived here and saw the idea of a lite-rpc for Solana, I thought it was interesting. but after I saw that I need to connect to an RPC for it to work I was confused. What would be the advantage of using lite-rpc since I need to connect to another RPC for it to work?

**Comment by ffarhanaamin** [comment]
My lite-rpc is connected to a RPC (public) node and I tried connecting it with the quick-forward-proxy but unfortunately, my quick-forward-proxy is not able to send over the transactions to TPU (I am not using a staked validator) and I keep getting these errors in the quic-proxy-log 

y::quinn_auto_reconnect: Failed connect initially to target 69.*.**11228    
2024-04-02T21:16:37.686454Z  WARN send_tx_batch_to_tpu: solana_lite_rpc_quic_forward_proxy::quinn_auto_reconnect: Failed connect initially to target 141.*.*.*:8013    
2024-04-02T21:16:37.686457Z  WARN send_tx_batch_to_tpu: solana_lite_rpc_quic_forward_proxy::quinn_auto_reconnect: Failed connect initially to target 141.98.216.131:8009  

IPs are masked just incase. 

Is it the normal behaviour or these ports are either normally filtered for any unwanted traffic? and in order to take advantage of the quic proxy do I need to run my own staked validator? 

Rust version: rustc 1.73.0 (cc66ad468 2023-10-03)
